crates/ironrdp-web/src/session.rs

```diff
@@ -165,6 +169,12 @@ impl SessionBuilder {
         self.clone()
     }
 
+    /// Optional
+    pub fn kdc_url(&self, kdc_url: String) -> SessionBuilder {
```

Comment:
Is it a KDC URL, or a KDC **Proxy** URL?
My understanding is that either we have a KDC address (`tcp://domain:88`, `udp://domain:88`) or a KDC Proxy URL (`https://gateway.devolutions.net/jet/KdcProxy?token=<…>`).
That is, when sending the messages via HTTP, it’s always using the KDC Proxy endpoint, with the payload wrapped in a [`KdcProxyMessage`](https://docs.rs/picky-krb/latest/picky_krb/messages/struct.KdcProxyMessage.html).
Unless I’m wrong, the naming should reflect this:
```suggestion
    pub fn kdc_proxy_url(&self, kdc_proxy_url: String) -> SessionBuilder {
```

Reply:
agreed, kdc_proxy_url makes more sense